examples/fast-incremental-builds/webpack.production.config.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
+
+module.exports = {
+    devtool: 'inline-source-map',
+    entry: './src/index.ts',
+    output: {
+        pathinfo: false
```

Comment:
That's `false` in production mode anyway. I guess you can omit it.

Reply:
Thanks @sokra